backend/tests/test_post_schedule_endpoint.py

```diff
@@ -0,0 +1,28 @@
+import server as serv
+from tools.create_db_for_tests import create_test_app_with_db
+import db.models as models
+
+
+def test_response_post_500():
+    models.db.drop_all()
```

Comment:
it would be better to have a fixture that creates a new database/transaction for each test requiring the database. 
Then you would not need to do cleanup manually in each test and the tests would be more predictable.

However it's ok as is for now.